database/redis/trigger_test.go

```diff
@@ -688,6 +710,53 @@ func TestTriggerErrorConnection(t *testing.T) {
 	})
 }
 
+func TestDbConnector_preSaveTrigger(t *testing.T) {
+	testTime := time.Date(2022, time.June, 6, 10, 0, 0, 0, time.UTC)
+
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+	systemClock := mock_clock.NewMockClock(mockCtrl)
+	systemClock.EXPECT().Now().Return(testTime).Times(4)
+	connector := &DbConnector{clock: systemClock}
+	patterns := []string{"pattern-1", "pattern-2"}
+
+	Convey("Test preSave function for local trigger", t, func() {
+		trigger := &moira.Trigger{ID: "trigger-id", Patterns: patterns}
+		Convey("Test preSave function for creation local trigger", func() {
```

Comment:
```suggestion

		Convey("Test preSave function for creation local trigger", func() {
```

Reply:
Check the official library example pls )

https://github.com/smartystreets/goconvey/blob/master/examples/bowling_game_test.go

Reply:
rewrote